# Position Sizing

Suppose there are X yens for me to do trading. The maximum loss I can take is 10% of the amount.<br />
If I plan to buy MetaPlanet(3350.T), and the average price is 880 yen. As the 20th Series of Stock Acquisition Rights has shown, the minimum exercising price is 777 yen. I adopt their strategy and calculate the maximum of loss per share.<br />

Therefore, the position sizing is:

In [4]:
def position_sizing(x):
    return 0.1 * x / (885-777)

# suppose the amount is 7000000 yen
display(position_sizing(7000000))

6481.481481481482

This (about 6481) is the number of shares I can buy.<br />
And I have to accept the loss if the share price becomes lower than 777.

总结：由此可以计算出仓位大小、让自己的损益在可控范围内。<br />
但按照近期股市走势来看， 虽然已经屯了2万的BTC了，但股价没有什么值得提振的点……原本靠机构行权融资还能保住股东利益，暂时不看好涨。<br />